python/paddle/v2/reader/creator.py

```diff
@@ -90,6 +93,7 @@ def recordio(paths, buf_size=100):
     """
     import os
     import paddle.v2.master.client as cloud
+    import cPickle as pickle
```

Comment:
Sorry, it's my mistake, we need `pickle.loads()` with the record fetched from master.
Done.